content/nomad/v1.11.x/content/docs/deploy/clusters/connect-nodes.mdx

```diff
@@ -214,6 +217,221 @@ consul {
 Refer to the [`consul` stanza] documentation for the complete set of configuration
 options.
 
+
+## Use client node introduction tokens
+
+You may restrict which clients join your cluster by configuring client
+introduction tokens. The client node introduction feature is like multi-factor
+authentication for your Nomad clusters. It does not replace mTLS but adds a
+second layer of security to prevent an unauthorized or misconfigured client from
+joining a Nomad cluster. 
+
+Each layer answers a distinct question: 
+
+- mTLS: Does the client have valid certificates for the cluster?
+- Client introduction token: Does the client have a valid token to join the
+cluster?
+
+You do not need to configure mTLS to use client node introduction tokens, but we
+do recommend securing your cluster with mTLS.
+
+You may optionally specify node names, node pools, and TTLs when you generate
+client introduction tokens.
+
+Follow these steps to use client node introduction tokens:
+
+1. [Create an ACL policy in which the node has write permissions](#create-an-acl-policy).   
+1. [Create an ACL role from the policy](#create-an-acl-role).
+1. [Generate a Nomad token](#generate-a-nomad-token).
+1. [Generate a client introduction token](#generate-a-client-introduction-token).
+1. [Configure the Nomad agent to use client introduction](#configure-the-nomad-agent).
+1. [Start the Nomad agent](#start-the-nomad-agent).
+1. [Monitor client join failures](#monitor-client-join-failures).
+
+### Create an ACL policy
+
+This example creates the `node:write` policy required to generate tokens.
+
+1. Create a policy file called `client-introduction.hcl`.
+
+   ```hcl
+   node {
+    policy = "write"
+   }
+   ```
+
+1. Create the policy.
+
+   ```shell-session
+   nomad acl policy apply client-introduction client-introduction.hcl
+   ```
+
+### Create an ACL role
+
+This example creates an ACL role called `client-introduction`.
```

Comment:
```suggestion
Create an ACL role, and attach the policy you created to the role.

This example creates an ACL role with the name `client-introduction` that uses the policy `client-introduction`.
```
1. Instructions to begin the section, then the example.
2. Trying to point out both CLI flags because the example uses the same name but that isn't a requirement.